qutebrowser/browser/hints.py

```diff
@@ -719,10 +719,21 @@ def start(self,  # pylint: disable=keyword-arg-before-vararg
             raise cmdexc.CommandError("No URL set for this page yet!")
         self._context.args = args
         self._context.group = group
-        selector = webelem.SELECTORS[self._context.group]
+        selector = self._get_selector()
         self._context.tab.elements.find_css(selector, self._start_cb,
                                             only_visible=True)
 
+    def _get_selector(self):
+        """Get the CSS selectors for this url and hinting group."""
+        group = self._context.group
+        selector = webelem.SELECTORS[group]
+
+        url = self._context.baseurl
+        group_name = webelem.Group(group).name
```

Comment:
Can't this just be `group.name`?

Reply:
Yep, turns out it can be. I'm used to C enums so I figured `group` would just be an int...